shuup/gdpr/resources.py

```diff
@@ -30,31 +34,51 @@ def valid_view(context):
     view_name = getattr(view_class, "__name__", "")
     if view_name == "EditorView":
         return False
+
     return True
 
 
 def add_gdpr_consent_resources(context, content):
     if not valid_view(context):
         return
 
-    shop = get_shop(context["request"])
+    request = context["request"]
+    shop = get_shop(request)
     gdpr_settings = GDPRSettings.get_for_shop(shop)
 
-    # GDPR not enabled
+    # GDPR not enabled, nothing to do
     if not gdpr_settings.enabled:
         return
 
+    # always add styles
+    add_resource(context, "head_end", static("shuup_gdpr/shuup_gdpr_styles.css"))
+
+    user = request.user
+    if not user.is_anonymous() and should_reconsent_privacy_policy(shop, user):
+        # TODO: Fix this as it may break when there are multiple consent documents
+        consent_page = get_privacy_policy_page(shop)
+        render_context = {
+            "request": request,
+            "csrf_token": context["csrf_token"],
+            "url": "/%s" % consent_page.url,
+            "accept_url": reverse("shuup:gdpr_policy_consent", kwargs=dict(page_id=consent_page.id))
+        }
+        update_resource = InlineMarkupResource(
+            loader.render_to_string("shuup/gdpr/privacy_policy_update.jinja", context=render_context)
+        )
+        add_resource(context, "body_end", update_resource)
+
     # consent already added
     # TODO: Should we check for changes and ask the consent again here?
```

Comment:
I think this was the point of this PR so maybe this TODO shouldn't exists either.